charts/airbyte/Makefile

```diff
@@ -0,0 +1,34 @@
+RELEASE_NAME := airbyte
```

Comment:
nit: I'd prefer for this to be a bash script instead of a `Makefile`. It'd be nice for people to be able to interface with this without installing xcode tools on Mac, for example.

Reply:
I can make that change. Do you have a preference on what the script is called? I assume you'd just want commands like the following?

```sh
./script-name install
./script-name test
./script-name update-docs
```

Reply:
@jrhizor, I moved this to a `ci.sh` bash script and removed the Makefile.